app/frontend/src/lib/ui/components/filterGroup.scss

```diff
@@ -89,7 +93,7 @@
       background: none;
       border: 1px solid govuk-colour("white");
       padding: 0;
-      border-bottom: 2px solid #b1b4b6;
+      border-bottom: 2px solid $govuk-border-colour;
```

Comment:
i used the colour b1b4b6 (which isnt a standard gov colour) as it was the same as the border of the tabs. personally i think should also change the tabs border to govuk-border-colour too

Reply:
As far as I can tell, the border of the tabs comes from here: https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/tabs/_index.scss

Which is alphagov's. And it does seem to use govuk-border-colour already, which translates to #b1b4b6.

Reply:
https://design-system.service.gov.uk/styles/colour/ govuk-border-colour is #b1b4b6

Reply:
oh does it, fair enough, maybe i didnt read gds colors through thouroughly